src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -250,13 +251,23 @@ UploadPanel.defaultProps = {
     name: 'LCUpload'
 };
 
+export function getUploadedFileName(){
+    var fields = FieldGroupUtils.getGroupFields(vFileKey);
+    const displayName = fields.rawTblSource.displayValue.split('\\');
+    return ( displayName && displayName[displayName.length-1])?displayName[displayName.length-1]:'';
```

Comment:
I tried to load two different files and did not notice the problem.  So I chose to do this way. 
 Another way that  I can think of  is to add a parameter in the request.  Then I need to modify the TableUtil.  I will discuss with you tomorrow.  Thanks!